docs/using/installing/index.rst

```diff
@@ -559,6 +559,19 @@ The configuration item to use AWS should look like:
 Make sure that the ``region`` and ``zone`` match each other and that both match the region and zone where the Flocker agent nodes run.
 AWS must be able to attach volumes created in that availability zone to your Flocker nodes.
 
+.. _emc-dataset-backend:
```

Comment:
Don't add unused references. However I think that these should be linked to by the Release Notes sentence which mentions them.